modules/azuredevops/project/backplane/documentation.tf

```diff
@@ -0,0 +1,45 @@
+# Add documentation template resource
+resource "azurerm_resource_group_template_deployment" "documentation" {
```

Comment:
wow I did not know

Reply:
is that common? maybe we add this to the AGENTS.md that this woul created each time